gen_local_test_certs.sh

```diff
@@ -1,6 +1,7 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
Can you test whether this works with dash and ksh? I believe the main reason why it was named bash and it was invoking explicitly bash is because it is using the non-portable `set -euxo pipefail`.

Reply:
You're correct, it doesn't work under Dash. I'm switching the interpreter back to Bash and updating `Dockerfile.local` to install Bash.